config/newrelic.yml

```diff
@@ -1,11 +1,15 @@
+common: &default_settings
+  host: <%= Rails.application.secrets.newrelic[:endpoint] %>
+
 <%= Rails.env %>:
+  <<: *default_settings
   app_name: <%= Rails.application.secrets.newrelic[:app_name] %>
   license_key: <%= Rails.application.secrets.newrelic[:license_key] %>
   log_level: <%= Rails.application.secrets.newrelic[:log_level] || 'info' %>
   monitor_mode: <%= Rails.application.secrets.newrelic[:enabled] ? 'true' : 'false' %>
-  apdex_t: <%= Rails.application.secrets.newrelic[:apdex_t] || 0.5 %>
```

Comment:
The `apdex_t` config has been deprecated and is now set in the New Relic UI, which is why this was removed.

Reply:
If you haven't done so already, please create a subtask for the cookbooks change and link it here. That should be tested in tandem with these changes.

Reply:
Related PR: https://github.com/GSA/usasearch-cookbooks/pull/665